charts/gateway-helm/templates/infra-manager-rbac.yaml

```diff
@@ -11,10 +11,12 @@ rules:
   resources:
   - serviceaccounts
   - services
+  - configmaps
```

Comment:
why is this needed ?

Reply:
For RateLimit

Reply:
dont we use xds for configuring intent ?

Reply:
https://github.com/envoyproxy/gateway/blob/958df4899d826c03ecd3aafaea379289bba85f3a/internal/infrastructure/kubernetes/ratelimit/resource_provider.go#L36

it's for statsd to prometheus configuration.

Reply:
ah